### DISCUSSION TITLE
Chinese Input Method Switching Issue in Translation Mode V1.3.3

### DISCUSSION THREAD

**Discussion by D-Simpackuser** [rustdesk/rustdesk#10118] [General]
### Bug Description

In v1.3.3, only 1:1 transfer mode,   the system indicates that the input method has switched to Chinese, it can't input anything, but in English input can work.

### How to Reproduce

1.Attempt to toggle the input method to Chinese.
2.Observe that the system displays the input method as "Chinese."
3.Attempt to input text, no word can be entered. and note that only English characters can be entered.

### Expected Behavior

The input method should correctly switch to Chinese and allow Chinese text input.

### Operating system(s) on local (controlling) side and remote (controlled) side

Windows 10 > Windows 10

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.1 > 1.3.1

### Screenshots

![image](https://github.com/user-attachments/assets/8961ebad-cc27-4fe7-8740-cd7ea79d3d2f)

**Comment by fufesou** [comment]
Both sides Chinese input method? You may also need to change the controlled side.

https://github.com/rustdesk/rustdesk/wiki/FAQ#keyboard-translation-modes

**Comment by D-Simpackuser** [comment]
That means if I need to use the Chinese input method on the controlled computer, I must downgrade to a version earlier than v1.3.3 and use the traditional mode instead of the 1:1 map mode.

**Comment by fufesou** [comment]
The desktop keyboard input is not changed. You need to switch to Input source 1.

<img width="218" alt="1732936827511" src="https://github.com/user-attachments/assets/d77c9a2d-88d1-4329-9b4b-daecd37f5c0a">

**Comment by D-Simpackuser** [comment]
Thank you very much! I have successfully implemented the functionality I wanted using the method you provided.
When I perform a fresh installation of version 1.3.3 (instead of upgrading from an earlier version), I can see all three control modes as expected. However, when upgrading to v1.3.3 from a lower version, only the 1:1 mode is displayed. I'm not sure if this is caused by something in the upgrade process.
